generated/niscope/enums.py

```diff
@@ -526,32 +503,32 @@ class TimingNCOFreqOffsetBits(Enum):
 class TriggerCoupling(Enum):
     AC = 0
     '''
-    AC coupled
+    AC coupling
     '''
     DC = 1
     '''
-    DC coupled
+    DC coupling
     '''
-    HF_REJECT = 3
+    HF_REJECT = 2
```

Comment:
It looks  like one of the APIs (CVI vs LabVIEW) has the wrong enum value. Please file an NI internal bug!

Reply:
Already filed: Internal CAR 675174